youtube_dl/extractor/rutube.py

```diff
@@ -44,6 +48,11 @@ class RutubeIE(InfoExtractor):
         'only_matching': True,
     }]
 
+    @classmethod
+    def suitable(cls, url):
+        parts = compat_urllib_parse_urlparse(url)
+        return super(RutubeIE, cls).suitable(url) and not parts.query
```

Comment:
This does not match https://rutube.ru/video/10b3a03fc01d5bbcc632a2f3514e8aab/?pl_id=4252.